kernel/libs/aster-bigtcp/src/socket/bound.rs

```diff
@@ -120,51 +368,177 @@ impl<E> AnyBoundSocketInner<E> {
         }
     }
 
-    pub(crate) fn is_closed(&self) -> bool {
-        match self.socket_family {
-            SocketFamily::Tcp => self.raw_with(|socket: &mut RawTcpSocket| {
-                socket.state() == smoltcp::socket::tcp::State::Closed
-            }),
-            SocketFamily::Udp => true,
+    /// Returns the next polling time.
+    ///
+    /// Note: a zero means polling should be done now and a `u64::MAX` means no polling is required
+    /// before new network or user events.
```

Comment:
This definition appears to be inconsistent with `poll_at`, which refers to the specific time for the next poll, and seems more aligned with `poll_delay`, which indicates the duration before the next poll. Wouldn't it be simpler to store the current time obtained from `get_network_timestamp`?

Reply:
A zero is definitely a time in the past, so I think it is acceptable to use it to indicate that a poll must be made immediately.

Otherwise, we should store `iface_cx.now()` instead of `get_network_timestamp`, since `iface_cx.now()` is cheaper and more consistent with other code in the polling logic.

Reply:
Thanks for your explanation! The current code is acceptable for me.